docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -341,6 +341,40 @@
     "sofa.upgrade_convention()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stream SOFA files\n",
+    "\n",
+    "SOFA files may be very large in which case you might not always want to read the entire file. To this end the `SofaStream` class can be used to\n",
+    "\n",
+    "- read parts of the data\n",
+    "- inspect the data and dimensions\n",
+    "\n",
+    "Lets stream some data from our first SOFA file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = os.path.join('..', '..', 'resources', 'my_first.sofa')\n",
+    "\n",
+    "with sf.SofaStream(filename) as file:\n",
+    "\n",
+    "    # get information about shape of the variable Data_IR\n",
```

Comment:
maybe also note that this is just copying the reference to data

Reply:
Not sure if I understand what you mean. Can you maybe make a suggestion?